src/v/cluster/security_frontend.cc

```diff
@@ -234,7 +234,9 @@ ss::future<std::vector<errc>> security_frontend::do_create_acls(
     if (should_sanction) {
         err = errc::feature_disabled;
         vlog(
-          clusterlog.warn, "Found Role-ACL bindings in request, ignoring...");
+          clusterlog.warn,
+          "An enterprise license is required to create Role-bound ACL "
```

Comment:
"Role-bound ACL" feels awkward
```suggestion
          "An enterprise license is required to create an ACL with a role binding"
```
But I see a similar message in `create_acls_handler`.

Reply:
Done